docs/install/harvester-configuration.md

```diff
@@ -425,3 +427,27 @@ install:
   vip_mode: dhcp
   vip_hw_addr: 52:54:00:ec:0e:0b
 ```
+
+### `system_settings`
+
+#### Definition
+
+This setting allows you to overwrite the default Harvester system settings.
+
+!!! note
+    Overwriting system settings is only applicable if Harvester is installed in "create" mode.
+    Instead, if the "join" mode is selected during the Harvester installation, this setting will
+    be discarded.
```

Comment:
```suggestion
    Overwriting system settings only works when Harvester is installed in "create" mode.
    If you install Harvester in "join" mode, this setting is ignored.
```

Reply:
In addition to the original words, I'd like to emphasize one thing (The bolded sentence):
> ..., this setting is ignored. **All the settings on the server ("create") Node will be applied to agent ("join") Nodes automatically.**

@nunix please also help check on this.

Reply:
@johnliu55tw we don't emphasize the difference between server or agent nodes a lot for normal users, so prefer to use the `create`  or `join` mode only.

Reply:
Good idea, I'll rephrase it a bit.